src/ExpressionEvaluator/Core/Test/FunctionResolver/Microsoft.CodeAnalysis.EE.FR.UnitTests.csproj

```diff
@@ -7,23 +7,23 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.ExpressionEvaluator.UnitTests</RootNamespace>
-    <AssemblyName>Roslyn.ExpressionEvaluator.FunctionResolver.UnitTests</AssemblyName>
+    <AssemblyName>Microsoft.CodeAnalysis.ExpressionEvaluator.FunctionResolver.UnitTests</AssemblyName>
```

Comment:
Your email stated the rule was "project file name must match assembly name" but this is still being violated here. Rule is more subtle than stated?

Reply:
@jasonmalinowski it's a bit more subtle. The suffix being UnitTests / IntegrationTests is hard and fast. The rest of the name differing is okay-ish at this point but could be changed later. @tmat can provide the details.

Reply:
It's simple: 
- .NET Core SDK defaults the assembly name to the project name, if not specified. It can be different if specified explicitly. 
- By convention, the toolset identifies categorizes projects as test projects if their name ends with `.UnitTests`, or `.Tests` (unit tests), or `.IntegrationTests` (for integration tests).